avro/src/main/java/com/fasterxml/jackson/dataformat/avro/deser/AvroParserImplDecoder.java

```diff
@@ -138,7 +138,14 @@ public void skipString() throws IOException {
     @Override
     public ByteBuffer readBytes(ByteBuffer old) throws IOException {
         consumeToken(JsonToken.VALUE_EMBEDDED_OBJECT);
-        return ByteBuffer.wrap(_parser.getBinaryValue());
+        byte[] value = _parser.getBinaryValue();
```

Comment:
How about I merge this separate from PR as it seems legit separate from other question.